vllm-speculative-decoding/config.yaml

```diff
@@ -0,0 +1,23 @@
+base_image:
+  image: nvcr.io/nvidia/pytorch:23.11-py3
+  python_executable_path: /usr/bin/python3
+build_commands: []
+environment_variables:
+  HF_TOKEN: ""
```

Comment:
why here over secrets?

Reply:
vLLM reads only this specific environment variable for the access token. It doesn't work with secrets